autoscale-docker-swarm-container/pom.xml

```diff
@@ -0,0 +1,253 @@
+                                    </exec>
+                                </entryPoint>
+                                <assembly>
+                                    <mode>tar</mode>
+                                    <inline>
+                                        <fileSets>
+                                            <fileSet>
+                                                <includes>
+                                                    <include>*.sh</include>
+                                                </includes>
+                                                <fileMode>0755</fileMode>
+                                            </fileSet>
+                                            <fileSet>
+                                                <includes>
+                                                    <include>*.yaml</include>
+                                                </includes>
+                                            </fileSet>
+                                            <fileSet>
+                                                <directory>src/main/config</directory>
+                                                <outputDirectory>config</outputDirectory>
+                                            </fileSet>
+                                        </fileSets>
+                                        <dependencySets>
+                                            <dependencySet>
+                                                <useProjectArtifact>false</useProjectArtifact>
+                                                <scope>runtime</scope>
+                                                <excludes>
+                                                    <exclude>com.github.krallin:tini</exclude>
+                                                </excludes>
+                                            </dependencySet>
+                                            <dependencySet>
+                                                <useProjectArtifact>false</useProjectArtifact>
+                                                <fileMode>0755</fileMode>
+                                                <includes>
+                                                    <include>com.github.krallin:tini</include>
+                                                </includes>
+                                            </dependencySet>
+                                        </dependencySets>
+                                    </inline>
+                                </assembly>
+                            </build>
+                            <run>
+                                <extraHosts>
+                                    <host>vagrant-mesos:192.168.56.10</host>
+                                </extraHosts>
+                                <env>
+                                    <DOCKER_HOST>http://${docker.host.address}:2375</DOCKER_HOST>
```

Comment:
there is no existing property I can find to get the port, so yes a change to output a property like ${docker.host.port} would be useful.
I'll raise a JIRA, but to be honest if we got unix:// socket support we could change the above to use the full docker_host information from the users machine.  This is just to work around this limitation.